packages/components/src/Collapsible/_docs/Collapsible.stories.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { Heading, Paragraph } from "@kaizen/typography"
+import { AddIcon } from "~components/Icons"
+import { Collapsible } from "../index"
+
+// eslint-disable-next-line storybook/prefer-pascal-case
+export const controlsSettings = {
+  children: { control: "disable" },
+  "data-testid": { control: "disable" },
+  sticky: { control: "disabled" },
+  open: { control: "boolean" },
+  noSectionPadding: { control: "boolean" },
+  lazyLoad: { control: "boolean" },
+  controlled: { control: "boolean" },
+}
```

Comment:
What's this for? Doesn't seem used.

Also, `export const` in a stories file will create a story. Probably not the intent?

Reply:
It was to have the controls in all stories align, but these can go